src/pure_pursuit.cpp

```diff
@@ -146,87 +150,90 @@ Point3D PurePursuit::get_lookahead_point( const Point3D& state )
 
 Point3D PurePursuit::get_point_on_path( const double& position )
 {
-	// constructor of this class initializes the path as m_robot_path (type Path (vector))
+	// constructor of this class initializes the path as m_robot_path (type Path (vector)
+	// )
 	// use the vector to find equation of a line and then from vect at 0
 	// this function simply returns the x,y, and z coordinate that represents the path
 	// blank position points from the beginning of the vector
 	double numerator, denominator;
-	double sum = 0;  // must be initialized to the first path point bc for loop adds on the following point
+	double sum = 0;  // must be initialized to the first path point bc for loop adds on
+	// the following point
 	double distance, slope = 0;
 	double zVal = 0, yVal = 0, xVal = 0;
 	unsigned long vectSize = m_robot_path.size();
 	Point3D newPoint;
 
 	if ( ( path_length() > position ) && ( position > 0 ) )
 	{
-		// loop until second to last point in path
-		for (uint32_t i = 0; i < vectSize - 1; i++ )
+		for ( unsigned long i = 0; i < ( vectSize - 1 ); i++ )
```

Comment:
this whole section has got some weird spacing. its hard to tell exactly what changed

Reply:
^^^^